lib/android-helpers.js

```diff
@@ -49,6 +49,15 @@ const CHROME_BROWSER_PACKAGE_ACTIVITY = {
 const SETTINGS_HELPER_PKG_ID = 'io.appium.settings';
 const SETTINGS_HELPER_UNLOCK_ACTIVITY = '.Unlock';
 const EMULATOR_PATTERN = /\bemulator\b/i;
+// These constants are in sync with
+// https://developer.apple.com/documentation/xctest/xcuiapplicationstate/xcuiapplicationstaterunningbackground?language=objc
+const APP_STATE = {
```

Comment:
nit: We could also use Object.frozen to make it immutable

Reply:
nit: I'm afraid but I cannot find `Object.frozen` function.
```
% node 
Welcome to Node.js v12.13.1.
Type ".help" for more information.
> const APP_STATE = Object.frozen({
...   NOT_INSTALLED: 0,
...   NOT_RUNNING: 1,
...   RUNNING_IN_BACKGROUND: 3,
...   RUNNING_IN_FOREGROUND: 4
... });
Thrown:
TypeError: Object.frozen is not a function
```

Reply:
Sorry, I meant https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze

Reply:
Oh, I see. Done https://github.com/appium/appium-android-driver/pull/699/commits/27fce44fb1cfede8050318c75c863b5a52544c8d